### PR TITLE
readme: add note about PowerShell command line options

### LINKED ISSUE #1769: Windows Powershell issue using Meson
When trying to configure Meson build targets the following command line was used and failed:

`meson configure -Dtargets=cortexm,stm`

The output was:

`PS D:\DataRoot\Projects\blackmagic\build> meson configure -Dtargets=cortexm,stm
ParserError: 
Line |
   1 |  meson configure -Dtargets=cortexm,stm
     |                                   ~
     | Missing argument in parameter list.
PS D:\DataRoot\Projects\blackmagic\build>`

The problem was traced to needing quotes around the target definition:

`meson configure "-Dtargets=cortexm,stm"`

While not a build system issue, this is worthy of a note when the Meson build system is documented.

### PR DESCRIPTION
**Pull request by perigoso**
On power shell some options like lists can cause issues and need to be wrapped in double quotes

## Detailed description

Adds a note to adress #1769

## Your checklist for this pull request

* [x] I've read the [Code of Conduct](https://github.com/blackmagic-debug/blackmagic/blob/main/CODE_OF_CONDUCT.md)
* [x] I've read the [guidelines for contributing](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md) to this repository
* [ ] ~~It builds for hardware native (see [Building the firmware](https://github.com/blackmagic-debug/blackmagic?tab=readme-ov-file#building-black-magic-debug-firmware))~~
* [ ] ~~It builds as BMDA (see [Building the BMDA](https://github.com/blackmagic-debug/blackmagic?tab=readme-ov-file#building-black-magic-debug-app))~~
* [ ] ~~I've tested it to the best of my ability~~
* [x] My commit messages provide a useful short description of what the commits do

## Closing issues

fixes #1769

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,6 +269,18 @@ When changing options after configuration, you may omit the argument `build` if
 
 You can have multiple build directories! So if you are regularly building firmware for multiple probes we would recommend keeping an individual build directory configured for each one.
 
+If you are working with PowerShell you may have some issue while trying to configure some options like the enabled target list `-Dtargets=cortexm,stm`:
+
+``` console
+PS C:\...\blackmagic\build> meson configure build -Dtargets=cortexm,stm ParserError: Line | 1 | meson configure build -Dtargets=cortexm,stm | ~ | Missing argument in parameter list. PS D:\...\blackmagic\build>
```
Comment (dragonmux) [inline README.md:275]: Does this correctly format over multiple lines, or are there some missing new lines here?

Think it wants to be:
```console
PS C:\...\blackmagic\build> meson configure build -Dtargets=cortexm,stm
ParserError:
Line | 1 | meson configure build -Dtargets=cortexm,stm
     | ~ | Missing argument in parameter list.
PS D:\...\blackmagic\build>
```

(if we've understood how PS formats this output)

**Review by dragonmux — COMMENTED** [review]
Looks good, we've got just one query around the PowerShell output block and then this can be merged.

File: README.md (hunk 1 above)
Comment (perigoso) [reply README.md]: You're right, I messed up the formatting. I'm not familiar with the formatting and don't have PS handy so I'll trust you got it right.

**Review by dragonmux — APPROVED** [review]
LGTM, we'll get this merged shortly. Thanks for the contribution!

**Pull request merged**